src/dev/build/tasks/clean_tasks.js

```diff
@@ -195,27 +195,21 @@ export const CleanExtraBrowsersTask = {
   async run(config, log, build) {
     const getBrowserPathsForPlatform = platform => {
       const reportingDir = 'node_modules/x-pack/plugins/reporting';
-      const phantomDir = '.phantom';
       const chromiumDir = '.chromium';
-      const phantomPath = p =>
-        build.resolvePathForPlatform(platform, reportingDir, phantomDir, p);
       const chromiumPath = p =>
         build.resolvePathForPlatform(platform, reportingDir, chromiumDir, p);
       return platforms => {
         const paths = [];
         if (platforms.windows) {
-          paths.push(phantomPath('phantomjs-*-windows.zip'));
```

Comment:
I wonder if we should leave this in for a release and remove in 7.x?  Otherwise I think developers will have phantom artifacts hanging around.

Reply:
Or maybe not, maybe the context is you can build for different OS's and clean up the previous artifacts, but you'll never have phantom hanging around. I'm a little hazy on this....

Reply:
Looks like this task just cleans a build environment, so that the build target doesn't have old dependencies. Yeah - leaving this alone would help w/ developers build Kibana in their environment. I think folks that do that usually will wipe out an entire stale build target directory. It's possible for someone to get no-longer used phantom binary, but I don't think it's critical.